dependencies.py

```diff
@@ -70,20 +74,81 @@
 logger = logging.getLogger(__name__)
 
 
+def prog_dialog(progress):
```

Comment:
Ik be nie z v/d afko's.
Waarom niet gewoon `progress_dialog`? (of `show_progress_dialog()` of `create_progress_dialog()`, ik weet niet wat 'ie precies *doet*.